setup.py

```diff
@@ -15,7 +15,7 @@
 
 setuptools.setup(
     name='wavefront-sdk-python',
-    version='1.1.2',
+    version='1.1.3',
```

Comment:
Should we hold on the version number, there's span log feature coming, want to be in the same version. Or probably using `.dev` thing?

Reply:
Is that something worth bumping the minor version to `1.2.0`, if not we can use `.dev` here for sure

Reply:
Cool, we decided to go with `1.2.0` for the span logs feature